### ISSUE TITLE
Tensorboard Image Summaries with TFLearn?

### ISSUE THREAD

**Issue by SinclairHudson** [tflearn/tflearn#1122]
Hello! I'm trying to see my feature maps in my neural network, or even just the kinds of outputs that my neural network is pumping out. I have no idea how to do this with tflearn though... What should I add to the following code so I can see some images in tensorboard?
`import tflearn
import datautil
import tensorflow as tf

batch_size = 64
testX, testY = datautil.getBatchRange(batch_size)

network = tflearn.input_data([None, 280, 120, 3], name='input')
network = tflearn.conv_2d(network, 3, 5, activation='leaky_relu')
network = tflearn.conv_2d(network, 3, 20,strides=[1,5,5,1], activation='leaky_relu')
network = tflearn.fully_connected(network, 336, activation='leaky_relu')
network = tflearn.reshape(network, (-1, 12, 28), name="output")
image = tf.reshape(network[:1], [-1, 12, 28, 1])
tf.summary.image("output", image)
network = tflearn.regression(network, optimizer='adam', learning_rate=0.05, # <--impatient
                     loss='categorical_crossentropy', name='target')

model = tflearn.DNN(network, tensorboard_verbose=3,
                    checkpoint_path="./RangeModel/", best_checkpoint_path="./Bestmodel/best.model", max_checkpoints=5)

for x in range(500):
    X, Y = datautil.getBatchRange(batch_size)
    model.fit({'input': X}, {'target': Y}, n_epoch=32,
              validation_set=({'input': testX}, {'target': testY}),
              snapshot_step=100, show_metric=True)
    model.save("RangeModel/"+str(x)+".model")`

**Comment by mhassandev** [comment]
@SinclairHudson if you have found the solution kindly let me know because i also want to display images in tensorboard using TFlearn

**Comment by j2233ack** [comment]
anyone know how to use?thx.
